frontend/ui/wikipedia.lua

```diff
@@ -768,9 +768,9 @@ time, abbr, sup {
 
     -- Done with adding files
     if cancelled then
-        progress_callback(_("Cleaning up..."))
+        progress_callback(_("Cleaning up…"))
```

Comment:
lol, what a cryptic way of saying I accidentally deleted an `n` :-P

I hadn't checked yet 'cause I assumed it was GitLab which caused all builds to fail yesterday. Anyway, should be good now.